versions/3.0.md

```diff
@@ -1152,6 +1152,51 @@ name: pet
 description: Pets operations
 ```
 
+#### <a name="examplesObject"></a>Examples Object
+
+Anywhere an `example` may be given, allow a $ref object.  This does mean that `example` can be either a string primitive or an object, like `additionalProperties`:
```

Comment:
This is a change to the 2.0 behavior, where the example property of a model is of the same type as what the model described. (I.e. `type: object` means it would be an object, `type: string` means it is a string, `type: number` means it is a number.) Now it seems to be a string containing formatted JSON (or a reference to such)?

If this is intentional, that needs to be updated too.